app/webpacker/components/ContactEditProfilePage/EditProfileForm.jsx

```diff
@@ -102,13 +100,10 @@ export default function EditProfileForm({
         onChange={handleFormChange}
         required
       />
-      <Form.Select
-        options={countryOptions}
+      <CountrySelector
         label={i18n.t('activerecord.attributes.user.country_iso2')}
-        name="country_iso2"
-        search
-        value={editedProfileDetails?.country_iso2}
-        onChange={handleFormChange}
+        selected={editedProfileDetails?.country_iso2}
```

Comment:
`selected` is not a supported property by `CountrySelector`. You should pass it as `countryIso2`